src/puget/printer.clj

```diff
@@ -170,6 +171,17 @@
 
 ;; ## Formatting Methods
 
+(defn- pos-int?
+  [x]
+  (if-let [core-pos-int? (resolve 'clojure.core/pos-int?)]
+    (core-pos-int? x)
+    (and (or (instance? Long x)
+             (instance? Integer x)
+             (instance? Short x)
+             (instance? Byte x))
```

Comment:
Could be simplified to use `integer?`, which does permit arbitrary precision values, but that's probably fine. Then it might make sense to call this function `pos-integer?` which would help avoid the name conflict with `pos-int?`.

Reply:
using `integer?` would give it different behavior to `clojure.core/pos-int?`

If using `integer?` feels better for `puget` in general, then we could replace this `(defn pos-int? ... (resolve 'clojure.core/pos-int?) ...)` here, along with its usages, with 
your own `(defn positive-integer? [x] (and (integer? x) (pos? x)))` and completely ignore the custom `resolve`, given that `integer?` and `pos?` were added in clojure 1.0

Reply:
Yeah, that's what I'm suggesting; you could even inline that in the only two call sites to `pos-int?`, instead of having a separate function defined for it. The logic in both places has to do with applying the `:coll-limit` setting when printing, so the exact type of the number is not important.

Reply:
sounds good; implemented it in https://github.com/greglook/puget/pull/55/commits/c2b7dc9c66fd0668a6e9ab4638603048c8022d81